package.json

```diff
@@ -17,12 +17,12 @@
     "build:fix-paths": "tscpaths -p ./tsconfig.json -s ./src -o ./lib",
     "circular-dependencies": "madge --circular src",
     "clean": "rimraf lib",
-    "coverage": "jest --coverage --silent",
+    "coverage": "vitest run --coverage --silent",
     "lint": "eslint ./src ./test ./types --color --ext .ts --ext .tsx",
     "postpack": "rm -f oclif.manifest.json",
     "prepack": "yarn clean && yarn build && oclif-dev manifest && oclif-dev readme",
     "prepare": "husky install",
-    "test": "FORCE_COLOR=0 jest",
```

Comment:
a difference here is that `vitest` command is in watch mode by default. `vitest run` will only run the tests once (and match the defaults in jest.